app/Models/Entry.php

```diff
@@ -376,7 +376,15 @@ public static function getContentByParsing($url, $path, $attributes = array(), $
 			CURLOPT_FOLLOWLOCATION => true,
 			CURLOPT_ENCODING => '',	//Enable all encodings
 		]);
+
 		curl_setopt_array($ch, FreshRSS_Context::$system_conf->curl_options);
+
+		if (isset($attributes['curl_params']) && is_array($attributes['curl_params'])) {
+			foreach ($attributes['curl_params'] as $co => $v) {
```

Comment:
Could you check whether `curl_setopt_array()` could be used there instead of a loop?